whelktool/scripts/cleanups/2022/01/lxl-3739-link-lcgft-terms-blank.groovy

```diff
@@ -0,0 +1,152 @@
+/*
+See LXL-3739 for more info.
+
+URI for <label> obtained from location response header of "curl -I -o headers -s ${url}"
+where url="https://id.loc.gov/authorities/genreForm/label/<label>"
+
+Manually edited (spelling mistakes etc):
+
+Darkwawe (music) -> Darkwave (Music) (in pm14bfp71ldshtn)
+Taraab -> Taraab (Music) (in wt7bjpmf1s02t5n)
+Occational verse -> Occasional verse (in khw087h30db2179)
+Alternative histories (fiction) -> Alternative histories (Fiction) (in 0xbdhl4j3g28rlz)
+Straight-edge -> Straight-edge (Music) (in 42gkrt7n4r3cqv8)
+Indians Music -> Indians--Music (in 86lpwfvs37m8dj2)
+Minutes (records) -> Minutes (Records) (in 0xbfnmfj240rck1)
+
+Terms not available at https://id.loc.gov/authorities/genreForm/label were checked manually with the LC
+suggest API https://id.loc.gov/suggest2?q=<label>. These are mainly music terms. See LABEL_TO_URI_NON_GF mapping.
+*/
+
+import whelk.util.DocumentUtil
+
+LABEL_TO_URI_NON_GF =
+        ["Aeolian harp"                      : "https://id.loc.gov/authorities/subjects/sh85001245",
+         "Ballets"                           : "https://id.loc.gov/authorities/subjects/sh85011293",
+         "Bowing (Musical instruments)"      : "https://id.loc.gov/authorities/subjects/sh2005003308",
+         "Christian literature, Early"       : "https://id.loc.gov/authorities/subjects/sh85025115",
+         "Concerts"                          : "https://id.loc.gov/authorities/subjects/sh85030620",
+         "Double bass"                       : "https://id.loc.gov/authorities/subjects/sh85039150",
+         "Guitar"                            : "https://id.loc.gov/authorities/subjects/sh85057803",
+         "Indians Music"                     : "https://id.loc.gov/authorities/subjects/sh85065058",
+         "Mass (Music)"                      : "https://id.loc.gov/authorities/subjects/sh85081852",
```

Comment:
"Masses" ?
https://id.loc.gov/authorities/genreForms/gf2014026926.html

Reply:
The usage here comes from https://libris.kb.se/katalogisering/pm14bd3742fkn0b where both "Mass (Music)" and "Masses" are used, so let's keep it that way and use both variants for now.